profile.md

```diff
@@ -0,0 +1,173 @@
+
+- *Service Broker*: Service Brokers manage the lifecycle of Services. Platforms
+  interact with Service Brokers to provision, bind and delete Service
+  Instances.
+
+- *Application*: The software that will make use of, be bound to, a Service.
+
+## Context Object
+
+In the [Open Service Broker API specification](spec.md) there are certain
+message flows that include a `context` property. This property is defined
+as an opaque JSON object that is meant to contain contextual information
+about the environment in which the Platform or Application is executing.
+For example, it might include the organizational information (eg. a
+Cloud Foundry `organization` GUID) in which the Application is owned.
+
+While the `context` property is defined as an opaque JSON object, in practice,
+it is often useful and necessary for there to be an agreed upon set of
+properties to ensure a common understanding of this data between the
+Platform and the Service Brokers.
+
+While use of this profile is OPTIONAL, an implementation is not compliant
+with this profile if it fails to satisfy one or more of the MUST, SHALL
+or REQUIRED level requirements defined herein.
+
+### Context Object Properties
+
+The list of properties within the Context Object can vary depending on
+which Service Broker API is being invoked and which Platform is being used.
+This section will define those properties and specify when each is meant
+to be used.
+
+Aside from the Platform specific properties, defined in the following
+sections, there is one common property called `platform` that
+MUST also appear within `context` to indicate which Platform is being used.
+
+The `platform` property MUST be a `string` and serialized as follows:
+```
+"platform" : "platform-string-here"
+```
+
+Each section below will define the `platform` value that MUST be used based
+on the Platform and the set of additional properties that MUST be present.
+
+#### Cloud Foundry
+
+*`platform` Property Value*: `cloudfoundry`
+
+The following properties are defined for usage within a Cloud Foundry
+deployment:
+
+- `organization_guid`<br>
+  The organization GUID as defined by the Cloud Foundry
+  specification/project. This property MUST be a non-empty string serialized
+  as follows:
+  ```
+  "organization_guid" : "organization-guid-here"
+  ```
+  For example:
+  ```
+  "organization_guid" : "1113aa0-124e-4af2-1526-6bfacf61b111"
+  ```
+
+- `space_guid`<br>
+  The space GUID as defined by the Cloud Foundry
+  specification/project. This property MUST be a non-empty string serialized
+  as follows:
+  ```
+  "space_guid": "space-guid-here"
+  ```
+  For example:
+  ```
+  "space_guid" : "aaaa1234-da91-4f12-8ffa-b51d0336aaaa"
+  ```
+
+The following table specifies which properties MUST appear in each API:
+
+Request API | Properties
+----------- | ----------
+`PUT /v2/service_instances/:instance_id` | `organization_guid`<br>`space_guid`
```

Comment:
I thought it looked better with it on a new line.  But if the group wants it....